src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -752,4 +752,15 @@ namespace winrt::TerminalApp::implementation
 
         args.Handled(true);
     }
+
+    void TerminalPage::_HandleGlobalSummon(const IInspectable& /*sender*/,
+                                           const ActionEventArgs& args)
+    {
+        args.Handled(false);
+    }
+    void TerminalPage::_HandleQuakeMode(const IInspectable& /*sender*/,
+                                        const ActionEventArgs& args)
+    {
+        args.Handled(false);
+    }

```

Comment:
```suggestion
    void TerminalPage::_HandleGlobalSummon(const IInspectable& /*sender*/,
                                           const ActionEventArgs& args)
    {
        args.Handled(true);
    }
    void TerminalPage::_HandleQuakeMode(const IInspectable& /*sender*/,
                                        const ActionEventArgs& args)
    {
        args.Handled(true);
    }
```

We want true right? That way we don't bubble this arg up any further?

Reply:
to be totally honest, these shouldn't _ever_ get here. But if we fail to register for the global hotkey, then returning false here will let the underlying terminal still process the key. Kinda like ctrl+c without a selection.